core/api/src/services/notifications/convert.ts

```diff
@@ -44,19 +44,17 @@ export const grpcNotificationCategoryToNotificationCategory = (
 ): NotificationCategory | InvalidPushNotificationSettingError => {
   switch (category) {
     case GrpcNotificationCategory.PAYMENTS:
-      return GaloyNotificationCategories.Payments
+      return NotificationCategory.Payments
     case GrpcNotificationCategory.PRICE:
-      return "Price" as NotificationCategory
+      return NotificationCategory.Price
     case GrpcNotificationCategory.CIRCLES:
-      return "Circles" as NotificationCategory
+      return NotificationCategory.Circles
     case GrpcNotificationCategory.BALANCE:
-      return GaloyNotificationCategories.Balance
+      return NotificationCategory.Balance
     case GrpcNotificationCategory.ADMIN_NOTIFICATION:
-      return GaloyNotificationCategories.AdminNotification
+      return NotificationCategory.AdminNotification
     default:
-      return new InvalidPushNotificationSettingError(
-        `Invalid notification category: ${category}`,
-      )
+      return new InvalidPushNotificationSettingError("Invalid notification category")
```

Comment:
why did you remove `${category}` also is this still only related to push notifications?

Reply:
Mistake, I'll add it back in